src/pages/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { PhotoList } from 'components';
+function SearchResults({ location }) {
+  const [photos, setPhotos] = useState([]);
+  const splitPath = location.pathname.split('/');
+  const searchTerm = splitPath[2];
```

Comment:
`const [,,searchTerm] = splitPath;` // you could try this

https://www.samanthaming.com/tidbits/13-skip-values-in-destructuring/